addons/mail/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -34,42 +37,42 @@
+                            <button class="o_ChatterTopbar_button o_ChatterTopbar_buttonAttachments btn btn-link rounded-0" type="button" t-att-disabled="chatter.isDisabled" t-on-click="chatter.onClickButtonAttachments">
                                 <i class="fa fa-paperclip"/>
                                 <t t-if="!chatter.isShowingAttachmentsLoading">
-                                    <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount" t-esc="chatter.thread ? chatter.thread.allAttachments.length : 0"/>
+                                    <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount pl-1" t-esc="chatter.thread ? chatter.thread.allAttachments.length : 0"/>
                                 </t>
                                 <t t-if="chatter.isShowingAttachmentsLoading">
-                                    <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-circle-o-notch fa-spin" aria-label="Attachment counter loading..."/>
+                                    <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-circle-o-notch fa-spin ms-1" aria-label="Attachment counter loading..."/>
                                 </t>
                             </button>
                             <t t-if="chatter.hasFollowers and chatter.thread">
                                 <t t-if="chatter.thread.channel_type !== 'chat'">
                                     <FollowButton
-                                        className="'o_ChatterTopbar_button o_ChatterTopbar_followButton'"
+                                        className="'o_ChatterTopbar_button o_ChatterTopbar_followButton rounded-0'"
                                         isDisabled="chatter.isDisabled"
                                         threadLocalId="chatter.thread.localId"
                                     />
                                 </t>
                                 <FollowerListMenu
-                                    className="'o_ChatterTopbar_button o_ChatterTopbar_followerListMenu'"
+                                    className="'o_ChatterTopbar_button o_ChatterTopbar_followerListMenu d-flex rounded-0'"
                                     isDisabled="chatter.isDisabled"
                                     threadLocalId="chatter.thread.localId"
                                 />
                             </t>
                         </div>
                 </div>
                 <t t-if="chatter.hasTopbarCloseButton">
-                    <div class="o_ChatterTopbar_buttonClose" title="Close" t-on-click="chatter.onClickChatterTopbarClose">
+                    <button class="o_ChatterTopbar_buttonClose flex-shrink-0 justify-content-center align-items-center d-flex rounded-lg-bottom bg-700 btn" title="Close" t-on-click="chatter.onClickChatterTopbarClose">
                         <i class="fa fa-times"/>
```

Comment:
You can replace it with `oi oi-large oi-close`. The raw css line tickling the font-size will not be necessary anymore